ush/global_cycle_driver.sh

```diff
@@ -18,9 +18,8 @@ pwd=$(pwd)
 export NWPROD=${NWPROD:-$pwd}
```

Comment:
@KateFriedman-NOAA Are NWPROD and DMPDIR recognized by the workflow?

Reply:
`DMPDIR` is, but only for dev runs to provide the location of the global dump archive (GDA). `NWPROD` has been removed from the workflow. I still see `NWPROD` or similar in a few components, which should likely also be cleaned out.

Reply:
Should I remove `NWPROD` under this issue or not bother?

Reply:
I'd remove it. I'd be ok with it going in this issue.

Reply:
Kate, I removed other unused variables. See 0a5e76a.